src/routes/Messages/Header/Header.js

```diff
@@ -54,9 +53,10 @@ export default class Header extends React.Component {
   render () {
     const { showAll } = this.state
     const { pending } = this.props
-    const otherParticipants = this.getOthers(this.props)
+    const participants = get('participants', this.props.messageThread) || []
+    const otherParticipants = this.getOthers(this.props.currentUser, participants)
```

Comment:
we generally like to pull out all the variables from this.props above, so add currentUser to const { pending } = this.props

Reply:
and messageThread too